RELEASE_NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Release Notes
+
+## Changes in PBFT 1.0.2:
+
+* Enable configuration of the storage location and max log size via the command
```

Comment:
Clarify "the command" (I assume that there are actually two). For example:
* Add ??? command to change the storage location
* Add ??? command to configure the max log size

Reply:
Updated